how-to/create-addon.rst

```diff
@@ -83,7 +83,7 @@ simply add it as the inner application directory.
 
     The *inner application directory*, in this case ``susan_example_application``, should have a
     name that matches the *package name* (``susan-example-application``), with underscores
-    substituted for the dashes.
+    substituting for the dashes.
```

Comment:
Was better previously, please revert this change

Reply:
Done